### ISSUE TITLE
SkinnedMesh + animations in multiple scenes

### ISSUE THREAD

**Issue by princemio** [mrdoob/three.js#7494]
hey,

i want to have an animated skinMesh and render it in two different scenes at the same time. One for physically emitting particles and the other one for displaying the mesh.
However if i do this, only one gets renderer. i know that this issue has been around and discussed - but since now the animation class gets updated i wanted to mention it again.

i found a super ugly workaround which is nothing else but heaving two different instances with copied values - but this cant lead to future results.

Could please someone help to find out why this is - and what can be done? 

thanks guys for reading!

mio

**Comment by tschw** [comment]
I don't quite get it. Are we talking about a rendering or an animation issue?

> i found a super ugly workaround which is nothing else but heaving two different instances with copied values - but this cant lead to future results.

You can animate once and post the results to multiple similar objects using `AnimationObjectGroup`.

**Comment by princemio** [comment]
hey guys - sorry missed that you guys replied. Ok the problem i have - can be best explained by the example

webgl_animation_skinning_blending.html
http://threejs.org/examples/webgl_animation_skinning_blending.html

now when i want to render my object to a scene i call
scene.add( blendMesh );

now it also works when i want to render it - to another Scene with Rendertarget
sceneRTT.add(blendMesh);

but when i want to render the same blendmesh object to two scenes 
                scene.add( blendMesh );
                sceneRTT.add(blendMesh);
then only one gets rendered...

I think the problem is Rendering two times the same animated blendmesh.

I can push anyother Threeobject to different renders and it works. But with that blendmesh i didnt get it.

Would be super nice if you guys could help me out. I think it offers a lot of possiblities. Not sure - about defered - but wouldnt this also be needed?

**Comment by tschw** [comment]
I guess you only have one `AnimationMixer` that is updated once per frame, so there should be no problem on the animation side.

Perhaps you have to force a matrix update? If that won't do, it's probably a renderer issue and you'll have to use two objects for now (see `AnimationObjectGroup` on `dev` to apply the same animation state to multiple objects).

**Comment by princemio** [comment]
Hey @tschw  - thank you man for looking into it. 

In my description of the problem - i used the example http://threejs.org/examples/webgl_animation_skinning_blending.html
So it should be possible to track the problem down - for someone beyond my horizont .hehe

The problem is that my objects are quite complex and it would be very redundant to upload two of them. I also think that this feature is very important as it enables to do physic calculation or create emitting maps in rendertargets ect. 

Basically the problem comes down to - if the first renderer does something to the animation - so the second cant access it. I think there is an architecture problem - please correct me if im off.

Thank you very much

**Comment by tschw** [comment]
This issue is about two renderers sharing an object - not really about animation. Ideally animation should run once (I guess that's how you do it?) and two renderers simply draw the resulting object (that's where something goes wrong). Do you have some code for us to reproduce the problem?

**Comment by WestLangley** [comment]
>but when i want to render the same blendmesh object to two scenes
scene.add( blendMesh );
sceneRTT.add(blendMesh);
then only one gets rendered...

An object can have only one parent, so when you add it to the second scene, it is removed from the first one.
